platform/android/export/export.cpp

```diff
@@ -778,6 +778,87 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
+			fa_metadata_text = "<meta-data android:name=\"com.oculus.vr.focusaware\" android:value=\"" + focus_awareness + "\"/>\n";
+			int dof_index = p_preset->get("xr_features/degrees_of_freedom"); // 0: none, 1: 3dof and 6dof, 2: 6dof
+			if (dof_index == 2) {
+				head_tracking = "<uses-feature android:name=\"android.hardware.vr.headtracking\" android:required=\"true\" android:version=\"1\"/>\n";
+			} else if (dof_index > 0) {
+				head_tracking = "<uses-feature android:name=\"android.hardware.vr.headtracking\" android:required=\"false\" android:version=\"1\"/>\n";
+			}
+			int hand_tracking_index = p_preset->get("xr_features/hand_tracking"); // 0: none, 1: optional, 2: required
+			if (hand_tracking_index == 2) {
+				hand_tracking = "<uses-feature android:name=\"oculus.software.handtracking\" android:required=\"true\"/>\n";
+			} else if (hand_tracking_index > 0) {
+				hand_tracking = "<uses-feature android:name=\"oculus.software.handtracking\" android:required=\"false\"/>\n";
+			}
+		}
+
+		String package_name = p_preset->get("package/unique_name");
+		String instrumentation = "<instrumentation\n"
+								 "        tools:node=\"replace\"\n"
+								 "        android:name=\".GodotInstrumentation\"\n"
+								 "        android:icon=\"@mipmap/icon\"\n"
+								 "        android:label=\"@string/godot_project_name_string\"\n"
+								 "        android:targetPackage=\"" +
+								 package_name + "\" />\n";
+
+		String manifest_chunk_1 = manifest_header + manifest_screen_sizes + manifest_gles + manifest_permission + head_tracking + hand_tracking + instrumentation;
+
+		//------------------------------------------------------------------------------------------
+
+		String application_header = "<application android:label=\"@string/godot_project_name_string\" "
+									"android:allowBackup=\"false\" tools:ignore=\"GoogleAppIndexingWarning\" "
+									"android:icon=\"@mipmap/icon\">)\n";
+
+		String plugins_names = get_plugins_names(get_enabled_plugins(p_preset));
+		String manifest_plugins;
+		if (!plugins_names.empty()) {
+			manifest_plugins = "<meta-data android:name=\"plugins\" android:value=\"" + plugins_names + "\"/>\n";
+		}
+
+		String manifest_activity = "<activity android:name=\"com.godot.game.GodotApp\" tools:replace=\"android:screenOrientation\" android:screenOrientation=\"landscape\">\n";
+		int orientation = p_preset->get("screen/orientation"); //default value is landscape
+		if (orientation == 1) {
+			manifest_activity = "<activity android:name=\"com.godot.game.GodotApp\" tools:replace=\"android:screenOrientation\" android:screenOrientation=\"portrait\">\n";
+		}
+
+		String manifest_chunk_2 = application_header + manifest_plugins + xr_metadata_text + manifest_activity + fa_metadata_text + "\n</activity>\n</application>\n</manifest>";
+		String manifest_text = manifest_chunk_1 + manifest_chunk_2;
+
+		String manifest_path = "res://android/build/src/release/AndroidManifest.xml";
```

Comment:
Please change to one of these two options:

`String manifest_path = p_debug ? "res://android/build/src/debug/AndroidManifest.xml" : "res://android/build/src/release/AndroidManifest.xml";`

```
String manifest_location = p_debug ? "debug" : "release";
String manifest_path = "res://android/build/src/" + manifest_location + "/AndroidManifest.xml";
```